source/sixtrack.f90

```diff
@@ -663,7 +663,7 @@ subroutine daten
 
   case("RIPP") ! Power Supply Ripple Block
     write(lerr,"(a)") "INPUT> ERROR RIPP module has been removed and replaced by DYNK."
-    write(lerr,"(a)") "INPUT>       The script rippconvert.py in the pytools folder can be used to convert the fort.3 file."
+    write(lerr,"(a)") "INPUT>       The script rippconvert.py in the pytools folder can be used to convert the input file."
```

Comment:
Is this message (pytools folder) still true?

Reply:
No idea. It's your script isn't it?

Reply:
I was just wondering, given the re-shuffling of all the files that have been going on.

```
15:28:26 kyrsjo@streifulv pytools $ ls
bdex_echo.py     fix_cdblocks.py  rippconverter.py  update_tests.py
compareSix2Madx  read_fortbin.py  SixRipken.py
15:28:26 kyrsjo@streifulv pytools $ pwd
/home/kyrsjo/code/SixTrack/source/pytools
```

Still there!

Reply:
Good!